### DISCUSSION TITLE
Is there any way to have transparent background

### DISCUSSION THREAD

**Discussion by kaustav202** [Ashutosh00710/github-readme-activity-graph#119] [General]
I've tried changing bg_color to transparent but it also removes the scale, values and axis of the graph

![act_graph_discuss](https://user-images.githubusercontent.com/89788120/167774610-5e208bfa-2749-43fb-8eef-85e9137b182e.png)

Are axes are part of the background itself?

**Comment by bijink** [comment]
Use new theme `github-compact` to achieve this style 
or 
by manually changing bg_color opacity to zero (eg: bg_color=00000000)

**Comment by AlphicJay** [comment]
> I've tried changing bg_color to transparent but it also removes the scale, values and axis of the graph
> 
> ![act_graph_discuss](https://user-images.githubusercontent.com/89788120/167774610-5e208bfa-2749-43fb-8eef-85e9137b182e.png)
> 
> Are axes are part of the background itself?

try changing opacity of main background color. For some odd reason i tried github-compact and ran into issue. maybe it's just me
